types/autoprefixer/package.json

```
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "postcss": "7.x.x"
+        "browserslist": "^4.15.x",
```

```suggestion
        "browserslist": "^4.15",
```
Also note, you have to open PR to MS tools, to allow install this via `package.json` dependency:
https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package